apps/admin/src/env.mjs

```diff
@@ -4,8 +4,8 @@ import { z } from "zod";
 
 export const env = createEnv({
     server: {
-        NEXTAUTH_URL: z.string().optional(),
-        NEXTAUTH_SECRET: z.string(),
+        APP_URL: z.string(),
+        HASH_SECRET: z.string(),
```

Comment:
Wenn ich hier in der Ansicht gerade nicht blind bin, fehlt die Umbenennung dieser env Variable in der env.sample auf globaler Ebene